mappings/net/minecraft/server/filter/TextFilterer.mapping

```diff
@@ -10,11 +10,14 @@ CLASS net/minecraft/class_5514 net/minecraft/server/filter/TextFilterer
 	FIELD field_26831 serverId Ljava/lang/String;
 	FIELD field_26832 ignorer Lnet/minecraft/class_5514$class_5515;
 	FIELD field_26833 executor Ljava/util/concurrent/ExecutorService;
+	FIELD field_36318 room Ljava/lang/String;
 	METHOD <init> (Ljava/net/URI;Ljava/lang/String;ILjava/lang/String;Ljava/lang/String;Lnet/minecraft/class_5514$class_5515;I)V
 		ARG 1 apiUrl
 		ARG 2 apiKey
 		ARG 3 ruleId
 		ARG 4 serverId
+		ARG 5 room
```

Comment:
```suggestion
		ARG 5 roomId
```
Also arg 7 can be named `parallelism` for how many CPUs this filterer is supposed to use